packages/lesmis/src/views/DashboardView.js

```diff
@@ -7,17 +7,149 @@ import React from 'react';
 import styled from 'styled-components';
 import MuiContainer from '@material-ui/core/Container';
 import Typography from '@material-ui/core/Typography';
+import { useUrlParams } from '../utils';
+import { useVitalsData } from '../api';
+import { EntityVitalsItem } from '../components';
 
-const TemplateBody = styled.section`
-  background: #e5e5e5;
-  padding-top: 1rem;
+const Wrapper = styled.section`
+  padding: 0, 0, 0, 0;
   min-height: 50vh;
+  background: #fbfbfb;
 `;
 
-export const DashboardView = () => (
-  <TemplateBody>
-    <MuiContainer>
-      <Typography>Dashboard View</Typography>
-    </MuiContainer>
-  </TemplateBody>
-);
+const Container = styled(MuiContainer)`
+  margin-left: 0;
+  padding: 0, 0, 0, 0;
+  position: relative;
+  display: flex;
+  flex-direction: row;
+  justify-content: space-between;
+  height: 100%;
+`;
+
+const WrapContainer = styled(MuiContainer)`
```

Comment:
```suggestion
const WrapContainer = styled(FlexRow)`
```

Best just to have one MuiContainer at the top level.

Reply:
From what I can tell `FlexRow` isn't a real type and is just a name used for `styled.div`s being used in a few places, I've swapped `WrapContainer` to be a `styled.div` called `FlexRow`